GVFS/GVFS.Platform.Mac/MacPlatform.cs

```diff
@@ -38,5 +39,33 @@ public override string GetDataRootForGVFSComponent(string componentName)
         {
             return new MacFileBasedLock(fileSystem, tracer, lockPath);
         }
+
+        public class MacPlatformConstants : GVFSPlatformConstants
+        {
+            public override string ExecutableExtension
+            {
+                get { return string.Empty; }
+            }
+
+            public override string InstallerExtension
+            {
+                get { return ".dmg"; }
+            }
+
+            public override string GVFSBinDirectoryPath
+            {
+                get { return Path.Combine("/", "usr", "local", this.GVFSBinDirectoryName); }

```

Comment:
nit: This seems like a weird use of Path.Combine with the "/" in the first argument. Can we just hard code this string?